src/Raven.Studio/typescript/common/notifications/notificationCenter.ts

```diff
@@ -61,6 +61,14 @@ class notificationCenter {
         this.recentErrors.dismissRecentError(alert);
     }
 
+    toggleNotifications() {
+         this.showNotifications.toggle();
+    }
+
+    notificationsState(state: boolean) {
+        this.showNotifications(state);
+        return true;
```

Comment:
Is it needed to return true? Maybe it is better to have 2 methods: showNotifications, hideNotifications. 
NotificationsStats as method name might be misleading. Additionally you won't need .bind(...., true/false) in views. 

Optionally you will have to call bindAll on notification center to bind `this`.